.ci/ansible/tasks/copy_test_files.yml

```
@@ -1,11 +1,21 @@
 ---
-- name: Create outputs dir
+- name: Create outputs dir (Linux)
   become: no
   ansible.builtin.file:
     path: "/home/{{ansible_user}}/e2e-testing/outputs"
     state: directory
     mode: '0755'
     owner: "{{ansible_user}}"
+  when: ansible_os_family != 'Darwin'
+
+- name: Create outputs dir (MacOS)
+  become: no
+  ansible.builtin.file:
+    path: "/Users/{{ansible_user}}/e2e-testing/outputs"
+    state: directory
+    mode: '0755'
+    owner: "{{ansible_user}}"
+  when: ansible_os_family == 'Darwin'
```

I think we can merge the Linux and MacOS task into one, using the `homePath` variable, and adjusting the when clause:
```suggestion
- name: Create outputs dir (Linux and MacOS)
  become: no
  ansible.builtin.file:
    path: "/{{ homePath }}/{{ ansible_user }}/e2e-testing/outputs"
    state: directory
    mode: '0755'
    owner: "{{ ansible_user }}"
  when: ansible_os_family in ["Darwin", "Linux"]
```